marklogic-data-hub/src/main/java/com/marklogic/hub/EntityManager.java

```diff
@@ -60,6 +60,10 @@
 
     boolean deployStagingQueryOptions();
 
+    HubEntity getEntity(String entityName);
+
+    HubEntity getEntity(String entityName, String version);
```

Comment:
One thing that's not obvious to me from a number of DHF methods is whether the entity is being retrieved from the project files or from the ML database. That's not specific to these methods, but I'm wondering if we could standardize on verbs that imply whether the data is retrieved from project files or a database. Or just make the method names more verbose - i.e. "getEntityFromProject" and "getEntityFromDatabase".

Reply:
I just noticed this calls "getEntities", which reads from the project. I think "getEntityFromProject" would be a good choice, with the notion that the "project" is a data store separate from the ML database.

Reply:
@rjrudin I believe that all of our managers read items from the project. We do make calls to the server to generate some of our items, like TDEs. I do think that could be a good practice to make it clear. Since these are new functions it would be a good place to start. I'll leave existing function signatures as is.